README.md

```diff
@@ -28,6 +28,20 @@ on unix platforms.
 
 ## Adding New SDKs
 
+### Order of actions
+
+1. update `.craft.yml` config to publish new package to your registry and GH
+2. cut the release
+3. update craft config to publish to release registry
+4. PR to this repository to manually add published new package. See steps below.
```

Comment:
The correct order is:

1. release without the craft modules (.craft.yml)
2. add the released version to the release registry manually
3. add the modules to craft modules (.craft.yml)
4. enjoy automatic releases

Reply:
Thanks, will update later

Reply:
There's work in progress to make this easier / unnecessary. So let's wait with this PR.